sw/device/lib/testing/test_framework/ottf_main.c

```diff
@@ -82,6 +87,13 @@ void _ottf_main(void) {
     LOG_INFO("Running %s", kOttfTestConfig.file);
   }
 
+  // Initialize a global random number generator testutil context.
```

Comment:
nit, maybe be a little more descriptive here
```suggestion
  // Initialize a global random number generator testutil context to provide tests with a source of entropy, e.g., for randomizing test behaviors.
```

Reply:
Done